vllm/engine/arg_utils.py

```diff
@@ -660,6 +661,14 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
             type=str,
             default=None,
             help='Target URL to which OpenTelemetry traces will be sent.')
+        parser.add_argument(
+            '--collect-model-forward-time',
+            type=bool,
+            default=EngineArgs.collect_model_forward_time,
+            help="If set to True and --otlp-traces-endpoint is set, "
+            "collects model forward time in traces. This involves "
+            "use of a blocking operation and hence might have a "
```

Comment:
can you actually test this? This is kind of interesting because tracing afaik is a production feature, but this will block it from being used in production...

Reply:
Yes, we have an internal setup at snowflake (where we collect these traces) and I do see the traces working including the model forward.